ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu-selector/gio-menu-selector.component.scss

```diff
@@ -34,19 +36,49 @@ $text: map.get(gio.$mat-dove-palette, default);
   }
 }
 
+.gio-menu-selector__mat-option.mat-option.mat-selected {
+  color: #000000; // TODO: Change this to $accent or $text...
+}
+
 ::ng-deep {
   gio-menu-selector {
     .gio-menu-selector {
       > * {
         display: block;
       }
 
-      @include mat.select-color(gio.$mat-theme-dark);
-      @include mat.form-field-color(gio.$mat-theme-dark);
-      @include mat.core-color(gio.$mat-theme-dark);
+      //@include mat.select-color(gio.$mat-theme-dark);
```

Comment:
I think we can fully delete this. If I understand correctly, it's `::ng-deep` but only for when there is a `mat-selector` in `gio-menu-selector`...